mock_event.go

```diff
@@ -0,0 +1,17 @@
+package hsm
```

Comment:
We should move the mock_ files to be suffixed with _test.go to avoid them being considered part of the public API of the package. If the are in _test.go files they will only be compiled and used when go test is run.

Reply:
Thanks, I'll make the change.